src/main/resources/assets/schemas/StudyPatch.yaml

```diff
@@ -0,0 +1,50 @@
+type: object
+properties:
+  name:
+    type: string
+    description: The name of the study.
+  description:
+    type: string
+    description: A human-readable description of the study.
+  studyType:
```

Comment:
Is StudyType enum centrally defined somewhere so that we can reuse and avoid duplication?